app/lib/tariff_synchronizer.rb

```diff
@@ -206,6 +206,8 @@ def rollback(rollback_date, keep: false)
               taric_update.delete
             end
           end
+
+          DataMigration.since(date_for_rollback).delete # Requeue data migrations
```

Comment:
WOW!

Reply:
So I can't believe this was missed as part of the rollbacks. It's basically a large advantage of the data migrator I'd imagined